src/main/java/IO/Main.java

```diff
@@ -0,0 +1,19 @@
+package IO;
+
+import java.io.*;
+
+public class Main {
+
+    private static final String PATH = System.getProperty("user.dir") + File.separator
+            + "src" + File.separator + "main" + File.separator + "resources" + File.separator;
+
+    private static String fileNameCSV = PATH + "CSVFile.csv";
+    private static String fileNameXLS = PATH + "XLSFile.xls";
+
+    public static void main(String[] args) throws IOException {
```

Comment:
not a best approach, commented in telegram group

Reply:
Fix